stablehlo/conversions/linalg/tests/BUILD.bazel

```diff
@@ -25,8 +25,8 @@ expand_template(
     testonly = True,
     out = "lit.site.cfg.py",
     substitutions = {
-        "\"@STABLEHLO_SOURCE_DIR@\"": "os.path.join(os.environ['TEST_SRCDIR'], 'stablehlo')",
-        "\"@STABLEHLO_TOOLS_DIR@\"": "os.path.join(os.environ['TEST_SRCDIR'], 'stablehlo')",
+        "\"@STABLEHLO_SOURCE_DIR@\"": "os.path.join(os.environ['TEST_SRCDIR'], '_main')",
```

Comment:
I'm not clear on why this change is needed. Can you expand on what this `_main` thing is?

Reply:
Tried to write in the PR description but I can see it's not very clear.
Basically Bazel 7 / bazelmod changed the layout directory structure such that the repo name is `_main` in the tree structure (https://bazel.build/remote/output-directories) and not the workspace name listed in `WORKSPACE`.

The ideal thing is to use the runfiles library in Python which handles all this.
`from python.runfiles import runfiles`

Reply:
Okay gotcha, makes sense. Thanks!

Reply:
Fly-by comment - this would break some of our downstream integrations pulling in stablehlo (and running its tests as part of downstream CI). Just a thought: Would adding `common --noenable_bzlmod` to the `.bazelrc` (like [this](https://github.com/cruise-automation/mlir-tcp/blob/main/.bazelrc#L48-L50)) resolve this, at least until stablehlo actually migrates to using bzlmod (depends on llvm migrating over as you clarified)?